### DISCUSSION TITLE
Welcome to Log4brains Discussions!

### DISCUSSION THREAD

**Discussion by thomvaill** [thomvaill/log4brains#1] [General]
## 👋 Welcome!

Since Log4brains is still in BETA, we primarily use Discussions to gather user feedback for now.

**So, if this is the first time you use Log4brains, please [open a new thread](https://github.com/thomvaill/log4brains/discussions/new?category=Feedback) a tell us what you think!**
Even if your feedback is negative, do not worry, I will really appreciate it since it helps us create a product that suits your needs!

We’re also using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by rberger** [comment]
Happy to have discovered Log4Brains! Really a great way to manage and share ADRs! Looking to make this core to our architecture design process.
